test/recipes/30-test_evp_data/evpciph.txt

```diff
@@ -259,31 +259,27 @@ Ciphertext = B2EB05E2C39BE9FCDA6C19078C6A9D1B
 #   AES-bits-CFB:key:IV/ciphertext':plaintext:ciphertext:encdec
 # CFB128-AES128.Encrypt
 Cipher = AES-128-CFB
-Availablein = default
```

Comment:
Should anyone try this with the legacy provider only, the test will fail miserably.  Therefore, instead of removing this line, I'd rather add another with `Availablein = fips`

Reply:
The other AES cipher modes (except OFB) don't include an `Availablein` line at all.  Why is CFB special?

Reply:
There is one instance of "Availablein = fips" in the EVP tests: to verify AES-XTS mode with duplicate keys errors out properly.